pkg/grpc/exec/exec.go

```diff
@@ -306,10 +315,9 @@ func GetProcessExit(event *MsgExitEventUnix) *tetragon.ProcessExit {
 		parent.RefDec()
 	}
 	if proc != nil {
+		// Copy process before returning
 		tetragonEvent.Process = proc.GetProcessCopy()
```

Comment:
Do we still need the copy then if we drop the UpdateEventProcessTid? We should be OK to just use the cached version then.

Reply:
yes done